src/parquet/column/test-util.h

```diff
@@ -316,7 +315,7 @@ template <typename Type>
 static shared_ptr<DictionaryPage> MakeDictPage(const ColumnDescriptor *d,
     const vector<typename Type::c_type>& values, const vector<int>& values_per_page,
     Encoding::type encoding, vector<shared_ptr<Buffer> >& rle_indices) {
-  InMemoryOutputStream page_stream;
+  InMemoryOutputStream page_stream(get_default_pool());
```

Comment:
All of these `get_default_pool` calls are very intrusive — why not set the default argument value to `nullptr` and use the default pool if it is not set by the user?